Frontend/Gmfctn/src/app/shared/models/dafault-user.ts

```diff
@@ -0,0 +1,13 @@
+import {User, Roles} from './user';
+
+export const defaultUser: User = {
```

Comment:
I don't think having this default user is a good approach.
As I see it exists only to fill the empty user variables, and as for me it's better to keep them empty as it'd be an indicator if the user is loaded or not

Reply:
Fixed